Automattic/pocket-casts-ios
# Settings Sync: Add files settings

| 📘 Part of: #1400 |
|:---:|

Adds settings for files "Auto Add to Up Next", "Delete Local Files" and "Delete Cloud Files" after playing.

<img src="https://github.com/Automattic/pocket-casts-ios/assets/3250/389d0d94-2db7-4e3e-ae27-90bd4f5215e5" width=300>

## To test

1. Log in with a Plus account on two devices/simulators (or enable plus in settings, although this will need to be done each time before you view settings - doesn't affect syncing).
2. D1: Navigate to Profile > Settings > Files
3. D1: Change one of the first three options, "Auto Add to Up Next", "Delete Local Files" or "Delete Cloud Files" after playing.
5. D1: Navigate back to the Profile and pull to refresh
7. D2: Pull to Refresh on Podcasts list or Profile
8. D2: Navigate to Profile > Settings > Files
9. D2: Verify that settings toggles show the proper values

## Checklist
